akka-stream/src/main/scala/akka/stream/javadsl/Sink.scala

```diff
@@ -148,9 +148,9 @@ class Sink[-In, +Mat](delegate: scaladsl.Sink[In, Mat]) extends Graph[SinkShape[
   def mapMaterialized[Mat2](f: japi.Function[Mat, Mat2]): Sink[In, Mat2] =
     new Sink(delegate.mapMaterialized(f.apply _))
 
-  def withAttributes(attr: OperationAttributes): javadsl.Sink[In, Mat] =
-    new Sink(delegate.withAttributes(attr.asScala))
+  override def withAttributes(attr: scaladsl.OperationAttributes): javadsl.Sink[In, Mat] =
```

Comment:
The usage of scaladsl.OperationAttributes in javadsl will be eliminated when OperationAttributes are unified in the other PR for #16951 that I have not yet submitted.